contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -98,6 +98,7 @@ extern bool pltsql_recursive_triggers;
 extern bool restore_tsql_tabletype;
```

Comment:
[General] Please follow commit description guidelines, we are missing `Task` and `signoff` fields. Use description of https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2748 as an example.